dashboard/app/views/print_certificates/show.html.haml

```diff
@@ -1,2 +1,4 @@
 - alt_text = @student_name ? "Certificate for #{@student_name}" : 'Certificate for Completion of One Hour of Code'
 %img{src: @certificate_image_url, alt: alt_text, width:"100%", style: "margin-top: 25px"}
+
+%script{src: webpack_asset_path('js/print_certificates/show.js')}
```

Comment:
I'm down with this approach! I was just looking through some haml this week and realized we didn't yet have an example of Amplitude events from dashboard haml files.